CleanUserAPI/Presentation/Controllers/UserController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Application.Interfaces;
+using Domain.Entities;
+using Infrastructure.JwtHandler;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Presentation.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly IUserRepo _userRepo;
+
+        private readonly IConfiguration _configuration;
+        private readonly ILogger<UserController> _logger;
+
+        public UserController(IUserRepo userRepo, IConfiguration configuration, ILogger<UserController> logger)
+        {
+            _configuration = configuration;
+            _userRepo = userRepo;
+            _logger = logger;
+
+        }
+
+        [HttpPost]
+        public IActionResult Login(Login user)
```

Comment:
could've used DTOs and called functions via a masked layer such as Service layer as it makes the code loosely coupled